modules/hydrodyn/src/HydroDyn_Output.f90

```diff
@@ -1006,9 +1006,10 @@ SUBROUTINE HDOUT_Init( HydroDyn_ProgDesc, OutRootName, InputFileData, y,  p, m,
    ! Check that the variables in OutList are valid      
    !-------------------------------------------------------------------------------------------------      
       
-   
-   CALL SetOutParam(InputFileData%OutList, p, ErrStat, ErrMsg )
-   IF ( ErrStat >= AbortErrLev ) RETURN
+   if (allocated(InputFileData%OutList)) then
```

Comment:
Not sure if this is related to Issue #1863 of Version 3.5.1.

Reply:
This is a good check to perform, just in case OutList hasn't been allocated, but it's not causing the referenced issue.

Reply:
This was mostly to fix issues in Debug mode where it would error due to passing an invalid array reference.